spec/factories/organization_news.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :organization_news do
+    news_article nil
+    organization nil
+  end
```

Comment:
Does this factory make sense? 

I recommend that you fix it and use it here: https://github.com/CodeCoreYVR/vancouvertechhub/pull/29/files#diff-e78a5546d3ead3a88b766a9041fa613cR10